### DISCUSSION TITLE
Old Data

### DISCUSSION THREAD

**Discussion by jp2nyy** [blaylockbk/Herbie#392] [Q&A]
How can I read in old historical model data such as Summer of 2015? I tried it once but it gave me an error. I am wondering as part of an experiment I want to do.

**Comment by blaylockbk** [comment]
It depends on what model you want to use and what is available.

HRRR data is available from 2014 to present on AWS, but other models are not archived for the same years. Looks like GFS on AWS goes back to 2021.

- https://aws.amazon.com/marketplace/pp/prodview-yd5ydptv3vuz2#overview
- https://aws.amazon.com/marketplace/pp/prodview-hok7o2o24ktfi?sr=0-1&ref_=beagle&applicationId=AWSMPContessa#resources


If Herbie doesn't find a forecast field, its because 1) the file doesn't exist, or 2) Herbie doesn't know where to look for it.

**Reply by jp2nyy** [reply]
I want to use historical NAM but from even before 2021.
